pytorch_blade/bazel/platform_alibaba/workspace.bzl

```diff
@@ -0,0 +1,3 @@
+
+def alibaba_workspace():
```

Comment:
Will this file be overwritten in internal repo?

Reply:
Yes, this will be overwritten.